src/loc.ml

```diff
@@ -70,6 +70,22 @@ let file_line path n =
       input_line ic
     )
 
+let file_lines path start stop =
```

Comment:
Since `start` and `stop` have the same type, what do you think about making them named for clarity?

Reply:
Good point. I'll use named arguments here